### ISSUE TITLE
All Pages - Poor Color Contrast of Top Banner Text

### ISSUE THREAD

**Issue by jfc3-dol** [presidential-innovation-fellows/worker-dol-gov#10]
While testing using the DHS Trusted Tester process, I found the color contrast of the SMALL paragraph text at top of page has a color contrast ratio of 3.28:1 and should be 4.5:1 or higher. Enlarging the text would also be helpful to those with low or poor vision to be able to read what it says.

Section 508 - 31(b)
WCAG 2.0 - 3.2 Color Contrast

**Comment by jfc3-dol** [comment]
Here is screenshot of poor color contrast.

![worker dot gov - poor color contrast of banner text medium gray and light gray background](https://cloud.githubusercontent.com/assets/10423487/19775582/e96889ac-9c3e-11e6-8e96-62923cdf7c17.png)

**Comment by adamsfallen** [comment]
Reverted to standard USWDS contrast level - 640676786f39a29c75c4eb87822ad3a0b9774c36

**Comment by jfc3-dol** [comment]
The color contrast now is good and has a color contrast ratio of  9.3:1, which is better than the 4.5:1 that is needed.

I there any way to make the text a bit bigger so people can read it?

**Comment by adamsfallen** [comment]
Also reverted to USWDS size - 7bc38e205c512221db68146e26cb53dea1234e8a

**Comment by jfc3-dol** [comment]
Making the text size a bit bigger should be helpful.

Closing issue.
